test/integration-ebpf/src/map_test.rs

```diff
@@ -14,7 +14,7 @@ static FOO: Array<u32> = Array::<u32>::with_max_entries(10, 0);
 #[map(name = "BAR")]
 static BAZ: Array<u32> = Array::<u32>::with_max_entries(10, 0);
 
-#[xdp]
+#[xdp(multibuffer = "true")]
```

Comment:
I think we shouldn't switch all tests to use frags. It's unlikely but just in case we end up breaking the non-frags case.

Reply:
ack. we still have 2 aya xdp programs in the integration tests that will ensure we support the non-frags case too... and any of the C progs used in integration tests are just plain old xdp